src/pages/credits/create/Create.tsx

```diff
@@ -174,17 +178,19 @@ export function Create() {
         )}
       </div>
 
-      <div className="my-4">
-        {credit && (
-          <InvoicePreview
-            for="create"
-            resource={credit}
-            entity="credit"
-            relationType="client_id"
-            endpoint="/api/v1/live_preview?entity=:entity"
-          />
-        )}
-      </div>
+      {(showPdfPreview === true || typeof showPdfPreview === 'undefined') && (
```

Comment:
I absolutely understand why you are commenting on this. David actually mentioned in the ticket, if we don't have this field, we have to take it as true. @turbo124 Please let me know if I got anything wrong in the map instructions.

@beganovich @turbo124 Perhaps we can just set this property to true by default on the API side to avoid this piece of code on the frontend side.

Reply:
@beganovich by default we should show the preview, however we will not have the settings configured for a new account.. Can default the interface prop to true ?

Reply:
Is this only for this setting? If not, we'd need abstraction on top of this, I don't think doing these checks everywhere makes any sense.

Reply:
I think it's only for this setting as far as I know. I have never done these checks on any other settings.
@turbo124 But David knows more details.

Reply:
We need abstraction on top of this. I suggest going with `useReactSettings` where you will provide default values.

Reply:
@beganovich The logic has been adjusted using useReactSettings hook. I hope that you agree with my logic.